presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -346,7 +346,10 @@ protected Void visitSingleColumn(SingleColumn node, Integer indent)
         @Override
         protected Void visitAllColumns(AllColumns node, Integer context)
         {
-            builder.append(node.toString());
+            if (node.getPrefix().isPresent()) {
+                builder.append("\"" + node.getPrefix().get().toString() + "\".");
+            }
+            builder.append("*");
```

Comment:
Change `AllColumns.toString` to either
* produce correct result (for formatting purpose), or
* produce something that's obviously not for formatting purpose